osx/build.sh

```diff
@@ -368,6 +369,17 @@ if [ $? -ne 0 ]; then
     exit 1
 fi
 
+((STEP++))
+echo "Updating KA Lite installer version"
+KA_LITE_VERSION="$(kalite --version)"
+DOCS_PATH="$SCRIPTPATH/KA-Lite-Packages/docs"
+sed -i '' "13s/0.17.0/$KA_LITE_VERSION/g" "$DOCS_PATH/INTRODUCTION.rst.rtfd/TXT.rtf"
```

Comment:
Here, you are replacing `0.17.0` - it would be better to change the files to have a placeholder like ``{{ KALITE_VERSION }}`` and replace this instead?